rtl/eclrtl/rtlbcd.hpp

```diff
@@ -82,16 +80,15 @@ ECLRTL_API char  DecClip(void *);
 ECLRTL_API void  DecRoundPos(void *,int by);
 ECLRTL_API void  SetMAccum(unsigned char c);
 
-ECLRTL_API void  DecLock();
-ECLRTL_API void  DecUnlock();
 ECLRTL_API unsigned DecMarkStack();
 ECLRTL_API void DecReleaseStack(unsigned mark);
 
+//No longer a critical section (since stack is thread_local), but prevents problems with exceptions.
```

Comment:
I didn't so that I didn't need to change the code generated by eclcc.

Actually.  I think I should leave dummy functions in the file - that throw errors.  Otherwise old workunits may fail to load, rather than reporting a version mismatch.